src/Apps/ArtQuiz/Hooks/useDislikeArtwork.ts

```diff
@@ -0,0 +1,20 @@
+import { graphql } from "react-relay"
+import { useMutation } from "Utils/Hooks/useMutation"
+import { useDeleteArtworkMutation } from "__generated__/useDeleteArtworkMutation.graphql"
+
+export const useDislikeArtwork = () => {
+  return useMutation<useDeleteArtworkMutation>({
+    mutation: graphql`
+      mutation useDislikeArtworkMutation($input: DislikeArtworkInput!)
+        @raw_response_type {
+        dislikeArtwork(input: $input) {
+          artwork {
+            id
+            slug
+            is_disliked: isDisliked
```

Comment:
We don't need to alias this. Assuming the old one was aliased for some unrelated compatibility issue that's no longer a problem.

Reply:
removed